smac/utils/validate.py

```diff
@@ -413,9 +413,13 @@ def get_runs(self, configs: Union[str, typing.List[Configuration]],
                     if len(inst_seed_config[i]) == 0:
                         inst_seed_config.pop(i)
                     # Add runs to runhistory
-                    for c in configs_evaluated:
+                    for c in configs_evaluated[:]:
```

Comment:
why?

Reply:
Configurations in `configs_evaluated` for this specific inst-seed pair contain runs that have been capped. They are removed in this for-loop, but when removing from the list we are iterating, python doesn't throw a warning but instead just skips elements. So the list is copied for iteration. I could use `tuple()` instead, so it's clear that an immutable object is desired?